ge0/parser.hpp

```diff
@@ -20,10 +23,9 @@ class Ge0Parser
   Ge0Parser();
 
   bool Parse(std::string const & url, Result & result);
-
-protected:
   bool ParseAfterPrefix(std::string const & url, size_t from, Result & result);
 
+protected:
   uint8_t DecodeBase64Char(char const c);
   static double DecodeZoom(uint8_t const zoomByte);
```

Comment:
nit: We are not used to writing const in function's integral params :)

Reply:
Are you asking me? I didn't write this code.